### PR TITLE
Added Config

### PR DESCRIPTION
**Pull request by MeNsaaH**

### CODE REVIEW

File: config.example.yaml [hunk 1]
```diff
@@ -4,28 +4,28 @@ hosts:
...
+      alias: home-server
+      connection: 
+        type: ssh
+        username: root 
+        password: somethingSecret
+        port: 33
+    "192.0.1.4":
+      connection: 
+        type: local
     eu-west1:
```
Comment (MeNsaaH) [inline config.example.yaml:21]: What's `eu-west1` supposed to be in this context?

Comment (MeNsaaH) [reply config.example.yaml:21]: @deven96 a group. You can create groups that share similar ssh configuration.

Comment (deven96) [reply config.example.yaml:21]: Why is it different from the group that you just put them directly under children

Comment (MeNsaaH) [reply config.example.yaml:21]: I don't understand

Comment (deven96) [reply config.example.yaml:21]: lol... nevermind na shayo I shayo that night of this comment

**Review by deven96 — APPROVED** [review]

**Pull request merged**